src/client/components/Modal/Modal.styles.css

```diff
@@ -0,0 +1,60 @@
+.modal-background {
+  background-color: rgba(0, 0, 0, 0.3);
+  position: fixed;
+  width: 100%;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.modal-box {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  padding-left: 80px;
+  padding-right: 80px;
+  padding-bottom: 25px;
+}
+
+.modal-background:active {
+  display: flex;
+}
+
+.modal-background:visible {
+  opacity: 1;
+}
```

Comment:
Don't understand when these states are used, can you explain?

Reply:
The opacity of the background wa effecting on modal so this was what I found in a website and made it work but I could see its working now even without this codes so it's deleted now.